views/job.hbs

```diff
@@ -0,0 +1,42 @@
+{{!< base.hbs}}
+
+<div class="row">
+    <section class="col-md-12 col-lg-12">
+        <h2>{{title}} at {{company}}</h2>
+        <h4>{{#if department }}{{department}} / {{/if}}<em>{{location}} {{#if is_remote}}(remote work available){{/if}}</em></h4>
+        <hr>
+    </section>
+</div>
+
+<div class="row">
+    <section class="col-md-8 col-lg-8">
+        <h3>Summary</h3>
+        {{{ summary.html }}}
+
+        <h3>Work Description</h3>
+        {{{ work_desc.html }}}
+
+        <h3>Requirements</h3>
+        {{{ requirements.html }}}
+
+        <h3>Why You Should Apply</h3>
```

Comment:
This section seems non-standard for job descriptions. Only concern that admin (ie you or me) could end up doing work to create it? Maybe should be optional (`{{# if}}`)